recipe/meta.yaml

```
@@ -11,45 +11,64 @@ source:
     - patches/0001-Patch-for-downstream.patch
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
 
 outputs:
   - name: coin-or-cbc
     script: build_package.sh
```

```suggestion
     script:
       -  build_package.sh  # [unix]
       -  build_package.bat  # [win]
```

Try something like this with the corresponding rename of the `bld.bat` file to `build_package.bat`